app/src/main/java/com/sdp/movemeet/Activity/Activity.java

```diff
@@ -335,6 +334,15 @@ public DocumentReference setBackendRef(DocumentReference newRef) throws IllegalA
         return backendRef;
     }
 
+    public String getDocumentPath() {
+        return this.documentPath;
+    }
+
+    public String setDocumentPath(String path) {
```

Comment:
How would we typically define the path ? Do you have an example of value it should take ?

Reply:
It would be set on Activity upload, you can call `.getDocumentPath()` once you have the `DocumentReference`